.github/workflows/deploy-testgrid.yaml

```diff
@@ -86,7 +87,26 @@ jobs:
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_PROD_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_PROD_SECRET_ACCESS_KEY }}
 
-    - run: docker push 799720048698.dkr.ecr.us-east-1.amazonaws.com/tgapi:${GITHUB_SHA:0:7}
+    - run: |
+        docker push 799720048698.dkr.ecr.us-east-1.amazonaws.com/tgapi:${GITHUB_SHA:0:7}
+
+  tgrun-image:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: tgrun-build
+        run: |
+          docker build -f testgrid/tgrun/Dockerfile \
+            -t replicated/tgrun:latest \
+            -t replicated/tgrun:${GITHUB_SHA:0:7} replicated/tgrun
+
+      - name: tgrun-push
+        env:
+          DOCKERHUB_DOCKER_PASS: ${{ secrets.DOCKERHUB_DOCKER_PASS }}
+          DOCKERHUB_DOCKER_USER: ${{ secrets.DOCKERHUB_DOCKER_USER }}
+        run: |
+           echo "$DOCKERHUB_DOCKER_PASS" | docker login -u $DOCKERHUB_DOCKER_USER --password-stdin
```

Comment:
https://github.com/replicatedhq/troubleshoot/blob/master/.github/workflows/build-test-deploy.yaml#L105

Better to use the docker login action for this please